charts/cluster-api-runtime-extensions-nutanix/README.md

```diff
@@ -32,12 +32,15 @@ A Helm chart for cluster-api-runtime-extensions-nutanix
 | deployment.replicas | int | `1` |  |
 | env | object | `{}` |  |
 | helmAddonsConfigMap | string | `"default-helm-addons-config"` |  |
-| helmRepositoryImage.pullPolicy | string | `"IfNotPresent"` |  |
-| helmRepositoryImage.repository | string | `"ghcr.io/nutanix-cloud-native/caren-helm-reg"` |  |
-| helmRepositoryImage.tag | string | `""` |  |
-| helmRepositorySecurityContext.fsGroup | int | `65534` |  |
-| helmRepositorySecurityContext.runAsGroup | int | `65534` |  |
-| helmRepositorySecurityContext.runAsUser | int | `65534` |  |
+| helmRepository.images.bundleInitializer.pullPolicy | string | `"IfNotPresent"` |  |
```

Comment:
nit: should we add a `helmRepository.enabled` instead of `selfHostedRegistry`?

Reply:
Done!

Reply:
this makes for a lot of changes and it breaks an existing API